packages/nightingale-navigation/README.md

```diff
@@ -4,45 +4,32 @@
 
 This custom element can be used to zoom and navigate along the sequence displayed in Nightingale. When a user interacts with the component, a `change` event is triggered which <nightingale-manager> uses to change `display-start` and `display-end` values on relevant custom elements.
 
-[Demo](https://ebi-webcomponents.github.io/nightingale/#/navigation)
-
 ## Usage
 
 ```html
 <nightingale-navigation
   length="456"
   display-start="143"
   display-end="400"
-  highlightStart="23"
-  highlightEnd="45"
-  rulerstart="50"
+  highlight="23:45"
+  rulerstart="1"
 />
 ```
 
 ## API Reference
 
-#### `length: number`
-
-The protein or nucleic acid sequence length.
-
-#### `display-start: number (optional)`
-
-The start position of the selected region.
-
-#### `display-end: number (optional)`
-
-The end position of the selected region.
+### Attributes
 
-#### `highlightStart: number (optional)`
+#### `rulerstart: number (default 1)`
```

Comment:
Naming, should it be "ruler-start" to go with the kebab-case approach (e.g. `display-start`)?